### PR TITLE
Updated Message text

### LINKED ISSUE #17541: Update message text in "Submission Held for Review" emails to developers
The body text for the template "submission held for review" emails to developers should be updated to the following: 

Hello, 

Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. 

If you have questions around your add-on review, please reply to the reviewer email or leave a reply on the Developer hub, Manage Status & Versions page.

If you are in urgent need of assistance not related to an add-on review, please contact amo-admins@mozilla.com

Thank you for your attention. 

--
Mozilla Add-ons
https://addons.mozilla.org

### PR DESCRIPTION
**Pull request by NegiAkash890**
Updated message text in "Submission Held for Review" emails to developers

Fixes  #17541

### CODE REVIEW

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 1]
```diff
@@ -1,7 +1,16 @@
 {% extends "reviewers/emails/base.ltxt" %}{% block content %}
-Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. Due to disruptions caused by the COVID-19 pandemic, it may take up to a week.
+Hello,
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:3]: The greeting is already in the base template. Please remove it (and the blank lines) to resolve the duplication.

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+If you have questions around your add-on review, please reply to the reviewer email or leave a reply on the Developer hub, Manage Status & Versions page.
+
+If you are in urgent need of assistance not related to an add-on review, please contact amo-admins@mozilla.com
 
 Thank you for your attention.
+
+--
+Mozilla Add-ons
+https://addons.mozilla.org
+
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:15]: Same here, this is already part of the base template.

**Review by wagnerand — CHANGES_REQUESTED** [review]

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {% extends "reviewers/emails/base.ltxt" %}{% block content %}
-Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. Due to disruptions caused by the COVID-19 pandemic, it may take up to a week.
-
-If you are in urgent need of assistance, please contact amo-admins@mozilla.com
 
+Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days.
+If you have questions around your add-on review, please reply to the reviewer email or leave a reply on the Developer hub, Manage Status & Versions page.
+If you are in urgent need of assistance not related to an add-on review, please contact amo-admins@mozilla.com
 Thank you for your attention.
+
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:7]: Remove this blank line.

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {% extends "reviewers/emails/base.ltxt" %}{% block content %}
-Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. Due to disruptions caused by the COVID-19 pandemic, it may take up to a week.
-
-If you are in urgent need of assistance, please contact amo-admins@mozilla.com
 
+Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days.
+If you have questions around your add-on review, please reply to the reviewer email or leave a reply on the Developer hub, Manage Status & Versions page.
+If you are in urgent need of assistance not related to an add-on review, please contact amo-admins@mozilla.com
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:5]: Add back blank line before this line.

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 5]
```diff
@@ -1,7 +1,8 @@
 {% extends "reviewers/emails/base.ltxt" %}{% block content %}
-Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. Due to disruptions caused by the COVID-19 pandemic, it may take up to a week.
-
-If you are in urgent need of assistance, please contact amo-admins@mozilla.com
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:2]: Remove this blank line.

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt [hunk 6]
```diff
@@ -1,7 +1,8 @@
...
 
+Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days.
+If you have questions around your add-on review, please reply to the reviewer email or leave a reply on the Developer hub, Manage Status & Versions page.
```
Comment (wagnerand) [inline src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:4]: Add back blank line before this line.

**Review by wagnerand — CHANGES_REQUESTED** [review]

**Comment by caitmuenster** [comment]
Hey @NegiAkash890, you're really close to getting this one landed! Do you want to finish it off?

**Comment by NegiAkash890** [comment]
> Hey @NegiAkash890, you're really close to getting this one landed! Do you want to finish it off?

Sure, I will this one tomorrow !

File: src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt (hunk 3 above)
Comment (NegiAkash890) [reply src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt:7]: Done!

**Comment by wagnerand** [comment]
Hi @NegiAkash890, we'd like to get this out this week, so I am taking over here. Thank you for getting it started.

**Pull request closed without merging**